tests/test_filter.py

```
@@ -1410,3 +1397,71 @@ def test_release_is_pack_false_for_last_final_season_movie_titles():
+    title = "Movie.2024.2160p.BluRay.x265-GROUP"
+    result = _make_result(title)
+    # Pre-seed a partial _meta missing the downstream-only fields.
+    result["_meta"] = {
+        "resolution": "1080p",  # deliberately wrong to prove a reparse happened
+        "hdr": [],
+        "audio": [],
+        "codec": "x264/AVC",
+        "languages": [],
+        "group": "GROUP",
+    }
+    filtered, all_parsed = filter_results([result])
```

Ruff (0.15.20)

[warning] Unpacked variable `filtered` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)